shadowsocks/servers.py

```diff
@@ -23,6 +23,8 @@
 import signal
 import time
 
+logging.basicConfig(filename=config['log-file'],level=logging.INFO)
```

Comment:
when i use this it worked
logging.basicConfig(filename=config.logfile,level=logging.INFO)
and  we nedd to add follow line to shadowsocks/shadowsocks/config.py
logfile = '/var/log/shadowsocks.log'